cmd/libs/go2idl/make-client/generators/client-generator.go

```diff
@@ -0,0 +1,270 @@
+/*
```

Comment:
For consistency with set-gen and deepcopy-gen, I would change the directory name from:
cmd/libs/go2idl/make-client/
to
cmd/libs/go2idl/client-gen/